api/src/main/resources/workbench.yaml

```diff
@@ -1269,6 +1269,32 @@ definitions:
         type: boolean
       emailVerificationStatus:
         $ref: "#/definitions/EmailVerificationStatus"
+      aboutYou:
+        type: string
+      areaOfResearch:
+        type: string
+      institutionalAffiliations:
+        type: "array"
+        items:
+          $ref: "#/definitions/InstitutionalAffiliation"
+
+  InstitutionalAffiliation:
+    type: object
+    required:
+      - userId
+      - institution
+      - role
+    properties:
+      institutionalAffiliationId:
+        type: integer
+        format: int64
+      userId:
+        type: integer
+        format: int64
```

Comment:
InstitutionalAffiliation is only returned under a user; no need to expose this here, either.